var_info.unified

```diff
@@ -44,7 +44,7 @@ BUILD_DEPENDS="debhelper (>= 7.0.50~),
  libgles2-mesa-dev [armel] | libgl1-mesa-dev | libgl-dev,
  libglew-dev,
  libglu1-mesa-dev | libglu-dev,
- libgnutls-dev,
+ libgnutls-deb0-28 | libgnutls28 | libgnutls26,
```

Comment:
Don't drop libgnutls-dev. And also add it to build depends section of control.unified as you had on the old PR.